signer.js

```diff
@@ -3,12 +3,20 @@
 const cookieSignature = require('cookie-signature')
 
 module.exports = function (secret) {
+  const secrets = Array.isArray(secret) ? secret : [secret]
+  const [signingKey] = secrets
+
   return {
     sign (value) {
-      return cookieSignature.sign(value, secret)
+      return cookieSignature.sign(value, signingKey)
     },
     unsign (value) {
-      return cookieSignature.unsign(value, secret)
+      for (const key of secrets) {
+        const result = cookieSignature.unsign(value, key)
+        if (result !== false) {
+          return result
+        }
+      }
     }
```

Comment:
The idea being that you can continue to decrypt old cookies, but new cookies are signed with the first key in the array?

Reply:
Yep, that's the intent here.
_I'm updating the docs_